### ISSUE TITLE
Add an option for inspecting DOM <=> Model/View

### ISSUE THREAD

**Issue by pomek** [ckeditor/ckeditor5-inspector#123]

**Comment by oleq** [comment]
What would it look like?

**Comment by magda-chrzescian** [comment]
It should provide a possibility to quickly find the place in the editor, that is being watched in the inspector at the given moment, and vice versa. It would useful especially for long documents, where it is hard to "synchronize" the editor content with the inspector.

So the basic functionality would be:
* Setting the selection in the proper place of the editor and scrolling to it when double-clicking on an element in inspector View/Model.
* Scrolling to the selection in the inspector when clicking some button in the inspector(?), like "Go to the selection".

Or maybe it would be enough to implement only the simpler of these solutions?

FYI @jacekbogdanski

**Comment by pomek** [comment]
I agree that looking for DOM elements based on our model/view can be annoying. CKEditor creates complex structures, and navigating into nested elements consumes some time. An option "Jump to DOM element" from model/view would be awesome.
